client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -255,14 +255,14 @@ const ProposalComment: m.Component<IProposalCommentAttrs, IProposalCommentState>
                 m(ProposalBodyEditMenuItem, {
                   item: comment, getSetGlobalReplyStatus, getSetGlobalEditingStatus, parentState: vnode.state,
                 }),
-                m(ProposalBodyDeleteMenuItem, { item: comment }),
-                parentType === CommentParent.Proposal // For now, we are limiting threading to 1 level deep
-                && m(ProposalBodyReplyMenuItem, {
-                  item: comment,
-                  getSetGlobalReplyStatus,
-                  parentType,
-                  parentState: vnode.state,
-                }),
+                // m(ProposalBodyDeleteMenuItem, { item: comment }),
+                // parentType === CommentParent.Proposal // For now, we are limiting threading to 1 level deep
+                // && m(ProposalBodyReplyMenuItem, {
+                //   item: comment,
+                //   getSetGlobalReplyStatus,
+                //   parentType,
+                //   parentState: vnode.state,
+                // }),
```

Comment:
why commented out?

Reply:
uhhhh I didn't touch this page, little confused why it's showing up in the changes. Could this be old code/artifact of a merge?